### PR TITLE
add .DS_Store to gitignore

### PR DESCRIPTION
**Pull request by dirkschulze**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ svgomg-code-coverage.html
 npm-debug.log
 test-summary.html
 tests/data-compare/*.svg
+.DS_Store
```
Comment (jaredadobe) [inline .gitignore]: +?

Comment (dirkschulze) [inline .gitignore]: Saying that this line was added. It is not in the actual file.

Comment (jaredadobe) [inline .gitignore]: Hah you know, this is the first time i have ever noticed they actually add it into the view. I am so used to their being tabs/spaces on the lines i am reviewing so it always looks like it is in a gutter!

Comment (jaredadobe) [inline .gitignore]: Looks good to me!

**Pull request merged**